index.html

```diff
@@ -53,5 +54,237 @@ <h1 >
         </div>
     </section>
 
+    <section id="work-section">
+        <h1>
+            <span id="work-heading">
+                My Recent<br>
+                Work
+            </span>
+        </h1>
+        
+        <section id="line">
+            <hr>
+        </section>
```

Comment:
- [ ] Kindly I recommend you to use `div` tag for instead of `section` tag for the new line to make your HTML code semantic correct.